pkgs/applications/networking/remote/teamviewer/default.nix

```diff
@@ -18,6 +18,7 @@ mkDerivation rec {
     tar xf data.tar.*
   '';
 
+  outputs = [ "out" "dev" ];
```

Comment:
By convention, `outputs` attribute goes between `pname`/`version` and `src`. See https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.